Makefile

```diff
@@ -24,7 +24,11 @@ OS_DIST ?= $(shell rpm --eval='%dist')
 
 PYTHON_VER ?= $(shell $(PYTHON) -c 'import sys; print("python%s.%s" % sys.version_info[:2])')
 
-PYTHON_SITELIB ?= $(PREFIX)/lib64/$(PYTHON_VER)/site-packages
+ifeq ($(filter-out debian ubuntu,$(OS)),)
+  PYTHON_SITELIB ?= $(PREFIX)/lib/$(PYTHON_VER)/site-packages
+else
+  PYTHON_SITELIB ?= $(PREFIX)/lib64/$(PYTHON_VER)/site-packages
+endif
```

Comment:
This looks ok.

Reply:
Using `$(OS_DIST)` variable now, because it seems more readable.

Debian distributions will have to set it from outside and since it is already so in the debian-build-instructions this should be save:
https://github.com/candlepin/subscription-manager/blob/32a9916998f99c99ddfc0469b004fd7132cfdf96/contrib/debian/rules#L21-L24

Reply:
OK :smiley: :+1: